src/python/zquantum/qaoa/farhi_ansatz.py

```diff
@@ -107,6 +107,6 @@ def get_symbols(self) -> List[sympy.Symbol]:
         """
         symbols = []
         for i in range(self.number_of_layers):
-            symbols.append(sympy.Symbol("beta_" + str(i)))
             symbols.append(sympy.Symbol("gamma_" + str(i)))
+            symbols.append(sympy.Symbol("beta_" + str(i)))
```

Comment:
That's actually not a good place to fix that – unless you also fix doctoring ;) 
I'd rather suggest leave it as it was, but change the code in line 98 and 99.